tests/test_2way_protocol.py

```diff
@@ -224,13 +242,23 @@ async def test_server_timeout(unused_tcp_port, no_tid_check, async_finalizer, po
     await agent.start()
     async_finalizer(agent.stop)
 
+    await assert_agent_counter(agent, 1, 0)
```

Comment:
This line should be moved after te wait until code, right?

Reply:
yes, thx